cocos/ui/UIScale9Sprite.cpp

```diff
@@ -503,23 +504,31 @@ namespace ui {
 
             auto capInsets = CC_RECT_POINTS_TO_PIXELS(_capInsetsInternal);
             auto textureRect = CC_RECT_POINTS_TO_PIXELS(_spriteRect);
-            auto spriteRectSize = CC_SIZE_POINTS_TO_PIXELS(_originalSize);
+            auto spriteRectSize = _spriteRect.size;
+            auto originalSize = CC_SIZE_POINTS_TO_PIXELS(_originalSize);
+            auto offset = CC_POINT_POINTS_TO_PIXELS(_offset);
+            
+            Vec4 offsets;
+            offsets.x = offset.x + (originalSize.width - textureRect.size.width) / 2;
```

Comment:
I don't quite understand the `offset.z` and `offset.w` mean, could you explain t? Thanks.